File: definitive/04_StructuredAPI/sc/snippets/overview.scala
import org.apache.spark.sql.types._


val df = spark.range(500).toDF("number")
df.select(df.col("number") + 10) // executed in Spark, not Scala


// Rows
spark.range(2).toDF().collect() // Row object


// Spark Types
val b = ByteType // Scala type

File: definitive/04_StructuredAPI/py/snippets/overview.py
from pyspark.sqpl.types import ByteType


df = spark.range(500).toDF('number')
df.select(df['number'] + 10)  # executed in Spark, not python


# Rows
spark.range(2).collect()  # Row object


# Spark types
b = ByteType()
